service/src/test/java/com/fitgoal/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,91 @@
+package com.fitgoal.service;
+
+import com.fitgoal.api.RegistrationService;
+import com.fitgoal.api.domain.User;
+import com.fitgoal.api.domain.UserRegistrationData;
+import com.fitgoal.api.exceptions.UserAlreadyExistException;
+import com.fitgoal.api.exceptions.UserNotFoundException;
+import com.fitgoal.dao.UserDao;
+import com.fitgoal.dao.domain.UserDto;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static com.fitgoal.service.util.TestHelper.createUserDto;
+import static com.fitgoal.service.util.TestHelper.createUserRegistrationData;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.any;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RegistrationServiceImplTest {
+
+    @Mock
+    private UserDao userDao;
+
+    private RegistrationService registrationService;
```

Comment:
same remarks about the initialization in the previous test

Reply:
Done